mlflow/gateway/constants.py

```diff
@@ -7,3 +7,6 @@
 # Provider constants
 MLFLOW_AI_GATEWAY_ANTHROPIC_MAXIMUM_MAX_TOKENS = 1_000_000
 MLFLOW_AI_GATEWAY_ANTHROPIC_DEFAULT_MAX_TOKENS = 200_000
+
+# Disallowed payload keys
+MLFLOW_AI_GATEWAY_DISALLOWED_QUERY_PARAMETER_KEYS = {"model"}
```

Comment:
I saw Azure OpenAI uses a parameter named engine as a model somewhere. Let me check.

Reply:
Never mind this.

Reply:
Found it:

<img width="777" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/a34f836b-e60b-4721-979c-30abed89ab12">

Reply:
I guess this doc is outdated?

Reply:
This doesn't matter because we don't use SDK.

Reply:
Yeah, the SDK internally adds these to the URL under the hood